### DISCUSSION TITLE
Freerdp version 3.3.0 runs in windows and the certificate cannot be read

### DISCUSSION THREAD

**Discussion by tao363** [FreeRDP/FreeRDP#10479] [Q&A]
After the compilation is successful, the freerdp client has been successfully generated, and when running FreerdP-proxy, it cannot read my certificate (the certificate is exported through mmc). In the crypto read pem function, the file is read to the pem error fread(pem,(size t)size,1,fp) after execution, the size of pem is 2046 but the size of fp is 2087. So the value of fr is zero！ How should I change it?

**Comment by akallabeth** [comment]
please try to describe your issue a bit better, I don´t understand what you try to ask.
(only thing I can suggest is you check the exported certificate is actually in `PEM` format, e.g. you can open it in a text editor and there is some readable text)

**Reply by tao363** [reply]
Thank you very much for your reply. It is the problem of the certificate. I changed the openssl version and then recompiled it
